features/application-title.yml

```
@@ -0,0 +1,9 @@
+name: application-title
```

We do this for the other meta tag feature. This is pretty new so I don't know how developers are going to recognize it. What do you think?

```suggestion
name: '<meta name="application-title">'
```